DigitalLearningSolutions.Data.Tests/Services/CustomPromptsDataServiceTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.Helpers;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class CustomPromptsDataServiceTests
+    {
+        private ICustomPromptsDataService customPromptsDataService;
+
+        [SetUp]
+        public void Setup()
+        {
+            var connection = ServiceTestHelper.GetDatabaseConnection();
+            customPromptsDataService = new CustomPromptsDataService(connection);
+        }
+
+        [Test]
+        public void GetCentreCustomPromptsByCentreId()
```

Comment:
Is it worth naming this test to be more "sentence-like" rather than just the name of the method (even though it is pretty basic)?

Reply:
yes, forgot to rename it